Library/src/Multiplayer/EventSerialisation.cpp

```diff
@@ -28,26 +28,30 @@ namespace
 {
 ESequenceUpdateType ESequenceUpdateIntToUpdateType(uint64_t UpdateType)
 {
+    ESequenceUpdateType SequenceUpdateType = ESequenceUpdateType::Invalid;
+
 	if (UpdateType == 0)
```

Comment:
Curious why we don't use a switch statement here? It's more efficient apparently.

Reply:
No good reason I know of. Will update to use a switch statement.

Reply:
Fixed in latest.